### PR TITLE
[NPU] Add minicpm-2b support for npu multi-processing

### PR DESCRIPTION
**Pull request by sgwhat**
### 1. Why the change?

Add minicpm-2b model support.

### 2. User API changes
```
-------------------- Input --------------------
<s> <用户>What is AI?
-------------------- Output --------------------
<s> <用户>What is AI?

AI, or artificial intelligence, is a field of computer science that focuses on creating intelligent machines that can perform tasks that typically require human intelligence, such as visual perception, speech recognition, and decision-making. These machines are designed to learn and improve over time, and they can be used in a variety of applications
```
### 3. Summary of the change 
- We disable `transpose_value_cache` make sure for the correct output.

### 4. How to test?
- [x] MTL
- [x] LNL

### CODE REVIEW

File: python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py [hunk 1]
```diff
@@ -123,18 +123,25 @@ def optimize_llm(
         modeling_module_name = model.__class__.__module__
         module = importlib.import_module(modeling_module_name)
 
+        if model.config.num_hidden_layers == 52:
+            # for minicpm-1b
+            transpose_cache = transpose_value_cache
+        elif model.config.num_hidden_layers == 40:
+            # for minicpm-2b
+            transpose_cache = False
```
Comment (jason-dai) [inline python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py:131]: Will it work on previous driver with `transpose_cache=true`?

**Comment by jason-dai** [comment]
Please also update the examples

File: python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py (hunk 1 above)
Comment (sgwhat) [reply python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py:131]: it could work.

Comment (sgwhat) [reply python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py:131]: it could work with previous driver on mtl, but cannot work on lnl.

**Review by jason-dai — APPROVED** [review]
LGTM

**Pull request merged**